### ISSUE TITLE
Not works anymore...

### ISSUE THREAD

**Issue by LonnyX** [pingpong-labs/menus#41]
Hi,

I imported your package but when I try to create a menu like this:

Menu::create('navbar', function($menu)
{
    $menu->url('home', 'Home', ['target' => 'blank']);
});

I have this error:
Non-static method Pingpong\Menus\Menu::create() should not be called statically, assuming $this from incompatible context

**Comment by nWidart** [comment]
You should import the menu facade, you used the menu class.

**Comment by LonnyX** [comment]
Already did... 

'Menu'      => Pingpong\Menus\MenuFacade::class,

and the provider

Pingpong\Menus\MenusServiceProvider::class,

**Comment by nWidart** [comment]
Yes, but in your error your example, you use the wrong one based of the error.

You need to use the **facade**

``` php
use Pingpong\Menus\Menu\MenuFacade as Menu

Menu::create('navbar', function($menu)
{
$menu->url('home', 'Home', ['target' => 'blank']);
});
```

**Comment by LonnyX** [comment]
It works now Thank you. PHPStorm doesn't import the right class :/ 

Thanx!

**Comment by nWidart** [comment]
No problem :smile: 

Yeah phpstorm couldn't know about the other classname. Issue is mainly the name of the facade in this package.

**Comment by LonnyX** [comment]
That's the disadvantage of using automated task :v:

**Comment by LonnyX** [comment]
PS: How do you make submenu ? something like this:

<ul>
   <li>home</li>
   <li>Account <ul><li>1</><li>2</li></ul></li>
</ul>

I see the getIcon method but how can I set the icon?
